quickwit/quickwit-cli/src/source.rs

```diff
@@ -337,7 +339,7 @@ async fn toggle_source_cli(args: ToggleSourceArgs) -> anyhow::Result<()> {
     let metastore = quickwit_metastore_uri_resolver()
         .resolve(&config.metastore_uri)
         .await?;
-    if args.source_id == INGEST_API_SOURCE_ID {
+    if args.source_id == INGEST_API_SOURCE_ID || args.source_id == CLI_INGEST_SOURCE_ID {
```

Comment:
We may want to align the naming of the IDs now that we're going to display both. Right now, we have `_cli-ingest-source` and `_ingest-api`. 

Your choosing ;)

Reply:
I am willing to go with  change `_ingest-api` -> `_api-ingest-source`

Reply:
Sounds good as well.